segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ConcatOperation.java

```diff
@@ -140,7 +140,7 @@ private void logEnd() {
         SLTS_CONCAT_LATENCY.reportSuccessEvent(elapsed);
         SLTS_CONCAT_COUNT.inc();
         if (chunkedSegmentStorage.getConfig().getLateWarningThresholdInMillis() < elapsed.toMillis()) {
-            log.warn("{} concat - late op={}, target={}, source={}, offset={}, latency={}.",
+            log.warn("{} concat - finished late op={}, target={}, source={}, offset={}, latency={}.",
```

Comment:
`finished late`  instead of just `late` simplifies grep search for successful calls.

Reply:
This kind of log messages actually flood the logs in many real-world experiments. I think that it would be good to reconsider this to be just `debug` level or remove this kind of messages completely (along with the `getLateWarningThresholdInMillis()` parameter), as we have metrics that report the latency values of operations. In fact, we have alerts inspecting such metrics that can notify us when the latency of storage operations is too high (easily configurable depending on the system at hand instead of being a system configuration parameter). Actually, with the current approach, we may have millions of `WARN` messages that may not be telling us nothing else than `getLateWarningThresholdInMillis()` is set too low for the system/workload at hand.

Reply:
+1. Make it debug. We really should keep the amount of INFO+ logging to a minimum.

This information can be extracted from metrics anyway - so really, why do you log it in the first place?